frontend/javascripts/oxalis/model/sagas/save_saga.ts

```diff
@@ -1132,7 +1151,13 @@ function* watchForSaveConflicts() {
   while (true) {
     const interval = yield* call(getPollInterval);
     yield* call(sleep, interval);
-    yield* call(checkForNewVersion);
+    try {
+      yield* call(checkForNewVersion);
+    } catch (exception) {
```

Comment:
Yes-ish. I'm not sure how exactly this escalated (I tried to debug the other issue which is why this might have been only a secondary effect). Then, I noticed that this could be a problem, if the client is offline for example. But you are right: Sending this to airbrake would be cleaner. I'll do so.